### PR TITLE
Apps Overview

### PR DESCRIPTION
**Pull request by guilhermedemouraa**

### CODE REVIEW

**Comment by guilhermedemouraa** [comment]
TODO: Take decent screenshots of the autoplot app by lunch tomorrow.

File: website/docs/apps/controller_app/README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+1. Click "Record" to start mapping your path.
+2. Drive the robot as desired.
+3. Click "Save" and name your track to use it later.
+4. Load the track when needed and click "Start Track" to initiate autonomous follow.
+
+### Remote Access via Web Browser
+
+The Autoplot is a web-app, and can be access through any web browser, such as Google Chrome.
+To view it in your browser, navigate to: `<element-vegetable>:8008`
```
Comment (mihelich) [inline website/docs/apps/controller_app/README.md:26]: We should note the prereq that they've [configured cross-network access](https://amiga.farm-ng.com/docs/ssh/#recommended-configure-cross-network-access) and added their web browser device to their Tailscale network.

Customers will need to use the FQDN form `[element-fruit].[tailnet-name].ts.net:8008`.

File: website/docs/apps/controller_app/README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+:::note NOTE
+Make sure to rename `<element-vegetable>` with your actual robot name.
+:::
+
+This app runs persistently in the background, even when not active on the robot's display,
+ensuring that you can control your Amiga from any web browser at any time.
```
Comment (mihelich) [inline website/docs/apps/controller_app/README.md:33]:
```suggestion
You can control your Amiga from a web browser at any time, even when the Autoplot app is not active on the Brain display.
```

"app runs persistently in the background" is a bit of implementation detail, and might not be the best way to describe it in the future.

**Review by mihelich — COMMENTED** [review]
Thanks @guilhermedemouraa, this is looking really good. Somewhat longer review as I think there are few things we should clarify for users.

File: website/docs/apps/filter_app/README.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+with the robot's frame, essential for precise navigation and operation.
+
+### Why Calibrate?
+
+- **Accuracy**: Proper calibration corrects any sensor biases and alignments,
+ensuring sensor readings are accurate.
+- **Performance**: Accurate sensor data is critical for effective task performance,
+especially in autonomous navigation.
+- **Maintenance**: Calibration is necessary when setting up your Amiga, after physical
+modifications, or when environmental factors could have affected sensor readings.
```
[the remark this answers is not in the document]
Comment (guilhermedemouraa) [reply website/docs/apps/filter_app/README.md]: Addressed in bca9e8979c5f220ec601aa06700e0e9eb23bd47d

File: website/docs/apps/filter_app/README.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+### How to Calibrate?
+
+1. **Check Calibration Status**: In the Filter App, look for the "Calibration Status" indicator.
+If it shows as "Calibrated" with a recent date, no further action is needed unless you've encountered
+issues that suggest recalibration is necessary.
```
[the remark this answers is not in the document]
Comment (guilhermedemouraa) [reply website/docs/apps/filter_app/README.md:28]: Addressed in bca9e8979c5f220ec601aa06700e0e9eb23bd47d

File: website/docs/apps/filter_app/README.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+- **Operational Precision**: More accurate settings lead to more precise path following.
+- **Adaptability**: Adjust settings when changing operational environments or the robot's physical setup.
+
+### How to Adjust Settings?
+
+1. **Access Settings Panel**: Click the "Settings" gear icon in the Filter App.
+
+2. **Adjust Parameters**:
+
+    a. Change GPS and IMU offsets to match your robot's setup.
```
[the remark this answers is not in the document]
Comment (guilhermedemouraa) [reply website/docs/apps/filter_app/README.md:71]: Addressed in bca9e8979c5f220ec601aa06700e0e9eb23bd47d

File: website/docs/apps/controller_app/README.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+---
+
+# Autoplot App Guide
+
+The Autoplot app is an advanced control interface for your Amiga, offering multiple modes of
+operation such as teach-and-repeat, WASD control, and remote access through a web browser.
+
+## How to Use the Autoplot App
+
+### Teach-and-Repeat
```
[the remark this answers is not in the document]
Comment (guilhermedemouraa) [reply website/docs/apps/controller_app/README.md:36]: Done here: 7bf35646

File: website/docs/apps/controller_app/README.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+- **S**: Disabled for safety
+
+Pressing these keys in your web browser will send commands directly to the robot, allowing for
+immediate control.
+
+:::info
+Using WASD control will override any ongoing autonomous tasks.
+:::
+
+## Autonomy Requirements
```
[the remark this answers is not in the document]
Comment (guilhermedemouraa) [reply website/docs/apps/controller_app/README.md]: Good point. I provided more information here: 7bf35646

File: website/docs/apps/controller_app/README.md (hunk 1 above)
Comment (guilhermedemouraa) [reply website/docs/apps/controller_app/README.md]: Note added here: 7bf35646

File: website/docs/apps/controller_app/README.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+
+Before starting any operation, make sure to check the following:
+
+- [ ] The robot is calibrated through the [**Filter App**](/docs/apps/filter_app).
+- [ ] The GPS antenna offsets are correctly set in the [**Filter App**](/docs/apps/filter_app).
+- [ ] Ensure the robot is connected to an RTK base station for enhanced GPS accuracy.
```
[the remark this answers is not in the document]
Comment (guilhermedemouraa) [reply website/docs/apps/controller_app/README.md]: Done here: 7bf35646

File: website/docs/apps/controller_app/README.md [hunk 9]
```diff
@@ -0,0 +1,172 @@
+---
+id: controller-app
```
Comment (Hackerman342) [inline website/docs/apps/controller_app/README.md:2]:
```suggestion
id: autoplot-app
```
+ where applicable & rename file `autoplot_app/README.md`

File: website/docs/apps/controller_app/README.md [hunk 10]
```diff
@@ -0,0 +1,172 @@
...
+- Monitor the robot's battery status via the Autoplot app to avoid interruptions in longer operations.
+
+## Troubleshooting
+
+If you encounter issues with autonomy or control:
+
+- Verify all status indicators in the Autoplot app's top bar are active and showing the expected states.
+- Check the [**Filter App**](/docs/apps/filter_app)
+to confirm that calibration and GPS settings are up-to-date.
+- If the robot does not respond to WASD commands, ensure that no other tasks are running and that
```
Comment (mihelich) [inline website/docs/apps/controller_app/README.md:169]:
```suggestion
- If the robot does not respond to WAD keyboard commands, ensure that no other tasks are running and that
```

**Review by mihelich — APPROVED** [review]
Thanks @guilhermedemouraa, this looks great.

And yeah, we should globally replace "Controller" references with either Autoplot app or Track Follower service. But that can be a separate PR.

File: website/docs/apps/controller_app/README.md (hunk 10 above)
Comment (mihelich) [reply website/docs/apps/controller_app/README.md:169]: I'll go ahead and commit this, and merge.

File: website/docs/apps/controller_app/README.md (hunk 9 above)
Comment (mihelich) [reply website/docs/apps/controller_app/README.md:2]: Let's do a follow-up PR to scrub controller references.

**Pull request merged**